libs/cluster/Session/ClusterCommands.cs

```diff
@@ -138,155 +138,86 @@ public void UnsafeWaitForConfigTransition()
+                    RespCommand.CLUSTER_FAILSTOPWRITES => NetworkClusterFailStopWrites(count, out invalidParameters),
+                    RespCommand.CLUSTER_FORGET => NetworkClusterForget(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_GOSSIP => NetworkClusterGossip(count, out invalidParameters),
+                    RespCommand.CLUSTER_GETKEYSINSLOT => NetworkClusterGetKeysInSlot(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_HELP => NetworkClusterHelp(count, out invalidParameters),
+                    RespCommand.CLUSTER_INFO => NetworkClusterInfo(count, out invalidParameters),
+                    RespCommand.CLUSTER_INITIATE_REPLICA_SYNC => NetworkClusterInitiateReplicaSync(count, out invalidParameters),
+                    RespCommand.CLUSTER_KEYSLOT => NetworkClusterKeySlot(count, out invalidParameters),
+                    RespCommand.CLUSTER_MEET => NetworkClusterMeet(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_MIGRATE => NetworkClusterMigrate(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_MTASKS => NetworkClusterMTasks(count, out invalidParameters),
+                    RespCommand.CLUSTER_MYID => NetworkClusterMyId(count, out invalidParameters),
+                    RespCommand.CLUSTER_MYPARENTID => NetworkClusterMyParentId(count, out invalidParameters),
+                    RespCommand.CLUSTER_NODES => NetworkClusterNodes(count, out invalidParameters),
+                    RespCommand.CLUSTER_REPLICAS => NetworkClusterReplicas(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_REPLICATE => NetworkClusterReplicate(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_RESET => NetworkClusterReset(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_SEND_CKPT_FILE_SEGMENT => NetworkClusterSendCheckpointFileSegment(count, out invalidParameters),
+                    RespCommand.CLUSTER_SEND_CKPT_METADATA => NetworkClusterSendCheckpointMetadata(count, out invalidParameters),
+                    RespCommand.CLUSTER_SETCONFIGEPOCH => NetworkClusterSetConfigEpoch(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_SETSLOT => NetworkClusterSetSlot(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_SETSLOTSRANGE => NetworkClusterSetSlotsRange(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_SHARDS => NetworkClusterShards(count, out invalidParameters),
+                    RespCommand.CLUSTER_SLOTS => NetworkClusterSlots(bufSpan, count, out invalidParameters),
+                    RespCommand.CLUSTER_SLOTSTATE => NetworkClusterSlotState(bufSpan, count, out invalidParameters),
+                    _ => throw new Exception($"Unexpected cluster subcommad: {command}")
```

Comment:
Is it possible to add a CLUSTER_INVALID case to deal e.g. cluster foo arg1 arg2 arg3?
I think we had support for it in main.

Reply:
I can put a specific error message back, but it'll go in the parsing layer.

Redis responds with `-ERR unknown subcommand 'foo'. Try CLUSTER HELP.` ; so I'll mimic that.